server/game/cards/08.5-TFM/SecretPact.js

```diff
@@ -0,0 +1,17 @@
+const DrawCard = require('../../drawcard.js');
+
+class SecretPact extends DrawCard {
+    setupCardAbilities(ability) {
+        this.attachmentRestriction(card => card.getType() === 'character' && !card.isFaction('martell'));
```

Comment:
Needs to be unique non-Martell character I think.

Reply:
Nope. "Uniquement" means "only" in this context.